packages/knowledge/src/base.ts

```diff
@@ -501,7 +501,7 @@ export const base: ChartKnowledgeBase = {
     shape: ['Round'],
     dataPres: [
       { minQty: 1, maxQty: 1, fieldConditions: ['Nominal', 'Ordinal'] },
-      { minQty: 1, maxQty: 1, fieldConditions: ['Interval'] },
+      { minQty: 1, maxQty: 2, fieldConditions: ['Interval'] },
```

Comment:
why？

I suppose nominal or ordinal could be 2 (one for color channel), but 2 interval fields?

Reply:
I think one interval field for angle and one interval field for length.

Reply:
length?  you mean 'width'?   I think the width of  segments is a constant，or determined by its hierarchical relationship.

Reply:
Oh, I confused it with the radial chart. So 2 nominal or ordinal fields, one for angle and one for color?

Reply:
1 interval field for angle channel, other fields (one or more than one, nominal or ordinals ) build a hierarchical relationship(color channel).

Reply:
Can the rose chart show a hierarchical relationship?

Reply:
Rose chart may not, its sectors  are equal angles.

Reply:
so, for rose chart shall we set here: 
```
dataPres: [
  { minQty: 1, maxQty: 2, fieldConditions: ['Nominal', 'Ordinal'] }, 
  { minQty: 1, maxQty: 1, fieldConditions: ['Interval'] }
],
```

Reply:
I agree.